app/src/main/res/values/googlepay_strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Do these need to be added to the main strings file for translations?

Reply:
No, Google says they should not be translated. https://developers.google.com/pay/api/android/guides/brand-guidelines#style